MCPTT/UI/Channel/ChannelList/ChannelListViewController.swift

```diff
@@ -45,38 +110,40 @@ extension ChannelListViewController: UICollectionViewDataSource, UICollectionVie
         guard let view = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionElementKindSectionHeader, withReuseIdentifier: "Header", for: indexPath) as? SectionHeaderCell else {
             return UICollectionReusableView()
         }
-        view.headerNameLabel.text = indexPath.section == 0 ? "Active" : "Idle"
+        view.headerNameLabel.text = indexPath.section == 0 ? "ACTIVE" : "IDLE"
```

Comment:
Why this change ? The previous implementation was as per the latest wireframes.

Reply:
I have reverted back these changes.